test/gpg-asym/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+. ../util.sh
+current_test=$(basename $(pwd))
+
+export LOCAL_DIR=$(mktemp -d)
+gpg-agent --daemon
```

Comment:
Just out of curiosity: why is this needed?

Reply:
Decrypting fails without it. But I think this can be avoided when adjusting the decryption cli command. I will look into it

Reply:
specifically, it is used for subsequent requests. e.g. importing the private key and then decrypting with it.

Reply:
While explicitly starting it is not required, the gpg agent is generally required in my understanding.. [1407661](https://github.com/offen/docker-volume-backup/pull/456/commits/14076614091ef0724756706d3e09ea363e1f2996)